cg/services/pacbio/metrics/models.py

```diff
@@ -14,8 +15,25 @@ class HiFiMetrics(BaseModel):
     median_read_quality: str = Field(..., alias=CCSAttributeIDs.MEDIAN_ACCURACY)
     percent_q30: float = Field(..., alias=CCSAttributeIDs.PERCENT_Q30)
 
-    @field_validator("percent_q30", mode="before")
-    def transform_percent_q30(cls, value: float) -> float:
-        if 0.0 <= value <= 1.0:
-            value *= 100
-        return value
+    _validate_percent_q30 = field_validator("percent_q30", mode="before")(fraction_to_percent)
+
+
+class ControlMetrics(BaseModel):
```

Comment:
Is it a Technical or experimental control?

Reply:
Maybe the class name can reflect that.

Reply:
Unclear. The file is called `control.report.json` and the attributes in our database class do not make any distinction on this matter. There is no other control specified anywhere, so I will leave it as it is